src/Fixer/Whitespace/NoTrailingWhitespaceFixer.php

```diff
@@ -70,15 +70,6 @@ public function getDefinition()
     /**
      * {@inheritdoc}
      */
-    public function getPriority()
```

Comment:
Then probably relocating the coment into \PhpCsFixer\AbstractFixer::getPriority will work (knowledge is preserved, no duplicated code)?